templates/templates.go

```diff
@@ -0,0 +1,467 @@
+// Code generated by go-bindata.
+// sources:
```

Comment:
This is missing the template files for the subfolders of templates/ such as CLA, .github, .gitlab, maintainer-examples etc.

Reply:
Yeah, I think I made a mistake when generated the templates.go file.